docs.md

```diff
@@ -394,6 +395,62 @@ export default deprecate('Do not use this component')(MyComponent)
 // When component is mounted, console will show warning: 'DEPRECATED: Do not use this component'
 ```
 
+## modal
+
+A function that returns a React HOC for creating modals.
+This HOC is dependent on the [`redux-modal`](https://github.com/yesmeck/redux-modal) library.
+
+The following functions are available on the wrapped component:
```

Comment:
Change to:
"The following functions are available as static properties on the wrapped component:"